src/main/java/com/tileman/TilemanModeMinimapOverlay.java

```diff
@@ -105,10 +105,12 @@ private void drawOnMinimap(Graphics2D graphics, WorldPoint point)
 	}
 
 	private Color getTileColor() {
-		if (plugin.getRemainingTiles() <= 0) {
-			return Color.RED;
-		} else if (plugin.getRemainingTiles() <= config.warningLimit()) {
-			return Color.ORANGE;
+		if(config.enableTileWarnings()) {
+			if (plugin.getRemainingTiles() <= 0) {
+				return Color.RED;
+			} else if (plugin.getRemainingTiles() <= config.warningLimit()) {
+				return Color.ORANGE;
```

Comment:
Let make this darker orange since it kinda looks like yellow right now.

Reply:
Changed colour